python/pyproject.toml

```diff
@@ -58,7 +58,7 @@ all = [
 ]
 
 [project.entry-points."aac"]
-rest-api = "rest-api"
+rest-api = "rest_api"
```

Comment:
Also, the file reference in the MANIFEST.in needs corrected to be `rest_api.aac`.

It's erroring out on my end because it doesn't know `rest-api` based on the dump I got 
```
      ValueError: invalid pyproject.toml config: `project.entry-points.aac.rest-api`.
      configuration error: `project.entry-points.aac.rest-api` must be python-entrypoint-reference
      [end of output]
```